src/components/EditorBottomBar.tsx

```diff
@@ -1,15 +1,20 @@
 import "./EditorBottomBar.css";
+import { useIntl } from "react-intl";
 
 interface EditorBottomBarProps {}
 
 export const EditorBottomBar = ({ ...otherProps }: EditorBottomBarProps) => {
+  const intl = useIntl();
+  const translate = (id?: any, values?: any) =>
+    intl.formatMessage({ id: id }, values);
+
   return (
     <div className="ed-cta-footer" {...otherProps}>
       <button type="button" className="dp-button button-medium secondary-green">
-        Salir y editar luego
+        {translate("bottombar_exit_button")}
```

Comment:
What do you think about simply _exit button_ or _exit_, it does not care where the button is.

```suggestion
        {translate("exit_button")}
```

Reply:
Maybe overlaps with topbar exit menu button

Reply:
But do not both buttons say the same?

Reply:
Nop:
`topbar_exit_button: 'Exit Editor'`
`bottombar_exit_button: 'Exit and edit later'`